From 2a275eeacbac9d58a01b3a24627acc82839b6444 Mon Sep 17 00:00:00 2001
From: Andrei Stefanescu <andrei.stefanescu@nxp.com>
Date: Wed, 3 Aug 2022 14:43:14 +0300
Subject: [PATCH 048/102] gpio: s32cc: remove reference to "s32cc-pinctrl.h"

That file was removed because it is no longer needed. Remove all
references to it.

Issue: ALB-8868

Upstream-Status: Pending 

Signed-off-by: Andrei Stefanescu <andrei.stefanescu@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 drivers/gpio/gpio-s32cc.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/gpio/gpio-s32cc.c b/drivers/gpio/gpio-s32cc.c
index 23a2c87793..ea6dd03e02 100644
--- a/drivers/gpio/gpio-s32cc.c
+++ b/drivers/gpio/gpio-s32cc.c
@@ -10,7 +10,6 @@
 #include <asm/io.h>
 #include <dm/pinctrl.h>
 #include <dt-bindings/gpio/gpio.h>
-#include <dt-bindings/pinctrl/s32cc-pinctrl.h>
 
 #define MSCR_OFF	0x0240
 #define GPDO_BASE	0x1300
-- 
2.17.1

